ui/main.qml

```diff
@@ -467,14 +492,27 @@ StatusWindow {
                 }
             }
             onCurrentPageNameChanged: Global.addCentralizedMetricIfEnabled("navigation", {viewId: currentPageName})
-        
+
             OnboardingStore {
                 id: onboardingStore
                 onAppLoaded: moveToAppMain()
                 onAccountLoginError: function (error, wrongPassword) {
                     onboardingLayout.stack.pop()
                 }
             }
+
+            Connections {
+                target: keychain
+
+                function onGetCredentialRequestCompleted(status, password) {
```

Comment:
I think we can implement this slot right inside the `Keychain` object, without the `Connections`?

Reply:
I think it's ok and clean because when Onboarding is unloaded we don't care about any signals from Keychain which may be used also in other places.